### PR TITLE
Feature responsive navbar

### PR DESCRIPTION
**Pull request by busratok**
### Description

- Responsive navbar is created
- Folder structure under the app folder changed according to new design:
- > fortalents folder is created
- > whatwedo page folder is moved under the fortalents folder as academy
- > jobandmore and jointhecommunity pages' folder is created under the fortalents folder

### CODE REVIEW

File: src/components/navbar/navbar.tsx [hunk 1]
```diff
@@ -1,59 +1,121 @@
...
 import logoOakyardOnDark from "../../../public/photos/logos/Logo_Oakyard_on_dark.svg";
 import Image from "next/image";
+import { AiOutlineMenu } from "react-icons/ai";
+import { AiOutlineClose } from "react-icons/ai";
+import { useState } from "react";
+
+const navlist = [
+  {
+    navName: "For Talents",
+    to: "#",
```
Comment (lucaCambi77) [inline src/components/navbar/navbar.tsx:13]: @busratok i see there is too much gap between `For talents`  and the `oakyard` logo. Can we reduce it sensibly? To me it looks kind of disjoint from the rest

Comment (busratok) [reply src/components/navbar/navbar.tsx:13]: Thank you for bringing this to my attention. I fixed it.

Comment (lucaCambi77) [reply src/components/navbar/navbar.tsx:13]: @busratok i still see some gap, for me say it should be an even distance both from the logo and the support us button.
Also, as you can see at the bottom, there are conflicts to solve

Comment (busratok) [reply src/components/navbar/navbar.tsx:13]: In the design file, there is no even distance. I put the gap according to it.

navbar-gap

Comment (lucaCambi77) [reply src/components/navbar/navbar.tsx:13]: ok it wasn't just looking good to me. We can change it later on if we want thanks @busratok

**Review by lucaCambi77 — APPROVED** [review]

**Pull request merged**